src/test/unit/java/com/commercetools/helper/mapper/impl/payment/DefaultPaymentMapperImplTest.java

```diff
@@ -72,11 +72,11 @@ public void ctpPaymentToPaypalPlus_withDiscount() throws Exception {
                 .isNull();
 
         assertThat(itemList.getItems().size()).isEqualTo(5);
-        assertItem(getItemBySkuQuantityPrice(itemList, "123454323454667", "1", "115.24"), "Halskette", "1", "115.24", "USD");
-        assertItem(getItemBySkuQuantityPrice(itemList, "123454323454667", "3", "115.24"), "Halskette", "3", "115.24", "USD");
+        assertItem(getItemBySkuQuantityPrice(itemList, "123454323454667", "Halskette", "1", "115.24"), "Halskette", "1", "115.24", "USD");
```

Comment:
Why is `Halskette` defined twice here?

Reply:
It is the name of the test Item. so, the string `"Halskette"` has used to equality check